integration-tests/src/test/java/oracle/kubernetes/operator/ItModelInImageConfigUpdate.java

```diff
@@ -415,6 +415,179 @@ public void testMiiConfigUpdateJdbcImage() {
     LoggerHelper.getLocal().log(Level.INFO, "SUCCESS - " + testMethodName);
   }
 
+  /**
+   * Create a domain without a JDBC DS using model in image and having configmap
+   * in the domain.yaml before deploying the domain. After deploying the domain crd,
+   * re-create the configmap with multiple model files that define a JDBC DataSource.
+   * Each model file overlaps the value of JDBC DS property oracle.net.CONNECT_TIMEOUT
+   * Update the domain crd to change domain restartVersion to reload the model,
+   * generate new config and initiate a rolling restart. The test verifies that loading order
+   * of model files follows WDT rules and the model file loaded last take precedence
+   */
+  @Test
+  public void testMiiMultiModelFilesLoadingOrderName() {
+    Assumptions.assumeTrue(QUICKTEST);
+    String testMethodName = new Object() {
+    }.getClass().getEnclosingMethod().getName();
+    boolean testCompletedSuccessfully = false;
+    final String connTimeout = "5203";
```

Comment:
no meanings. I just used the default number 5200 from mii sample and added 1 for each files

Reply:
can't find the sample I copied from :(   I was just thinking to make a diff value of connTime for each model file so that the test can verify that config update of connTime succeeded

Reply:
added the comments below:

// The value for property oracle.net.CONNECT_TIMEOUT,in milliseconds,
// for a client to establish an Oracle Net connection to the database instance.